site/docs/pages/xmtp/introduction.mdx

```diff
@@ -1,13 +1,45 @@
 ---
 title: Introduction to XMTP Kit · OnchainKit
-deescription: Introduction to XMTP Kit
+description: Introduction to XMTP Frames
 ---
 
 # Introduction to XMTP Kit
 
-XMTP Kit is designed to be modular and flexible, allowing developers to use only the parts of the protocol that they need.
+[XMTP](https://xmtp.org/) is a wallet to wallet messaging protocol, and many XMTP client applications are adding Frames support.
```

Comment:
`and many XMTP client applications` this information is not required to a developer. Let's keep it streight to the point.